docs/source/technical_tutorials/access/lpdaac_gedi_access.ipynb

```diff
@@ -0,0 +1,258 @@
+{
```

Comment:
<p>Should we be doing a download here? Or just show loading or reading the data directly from S3 with h5py and/or xarray?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/334/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Could switch it up, I was just keeping it consistent with the other "accessing" tutorials

Reply:
But this isn't like other tutorials, avoiding downloading is the point of direct s3 access.